selfdrive/controls/lib/pathplanner.py

```diff
@@ -119,14 +119,10 @@ def update(self, sm, pm, CP, VM):
         if not one_blinker or below_lane_change_speed:
           self.lane_change_state = LaneChangeState.off
         elif torque_applied:
-          self.lane_change_state = LaneChangeState.laneChangeStarting
-
-      # starting
-      elif self.lane_change_state == LaneChangeState.laneChangeStarting and lane_change_prob > 0.5:
-        self.lane_change_state = LaneChangeState.laneChangeFinishing
+          self.lane_change_state = LaneChangeState.laneChangeFinishing
 
       # finishing
-      elif self.lane_change_state == LaneChangeState.laneChangeFinishing and lane_change_prob < 0.2:
+      elif self.lane_change_state == LaneChangeState.laneChangeFinishing and lane_change_prob < 0.5:
```

Comment:
How did you find the 50%? Or is the dropoff so sharp it doesn't matter?

Reply:
Well that's when the model is more than 50% sure lane change is over. But it doesn't really matter anyway since this is only for UI, doing another lane change.